### ISSUE TITLE
BUG:  free(): double free detected in tcache 2

### ISSUE THREAD

**Issue by cesarmarcanove** [taviso/123elf#97]
![bug shaphot lotus 123 linux 2022-07-07 16-13-43](https://user-images.githubusercontent.com/279859/177863893-170f1e0e-8630-4a36-88a1-d91b1dd67e75.png)

**Bug discovered:**

                `free(): double free detected in tcache 2`

**PD:** Don't forget to launch RC2, current launched as RC1.
             Hopefully theywill release it to other platforms such as: Android, Windows (including 64 bits), FreeBSD, OpenBSD, Minix 3.4.x (or newer), Arch Linux, etc. The creator of Lotus 123 is same who created the know webbrowser Mozilla Firefox.

**Comment by taviso** [comment]
Thank you for the bug report!

Do you know what you were doing at the time that caused it?

**Comment by cesarmarcanove** [comment]
> Thank you for the bug report!
> 
> Do you know what you were doing at the time that caused it?

Yes. I was finishing after calculating, I saved it and exited the app, suddenly a "double cache" bug appears that I would have detected.

**Comment by taviso** [comment]
That's not much to go on, but I'll see if I can spot the problem!
